internal/planning/deploy/deploy.go

```diff
@@ -272,7 +274,7 @@ func prepareBuildAndDeployment(ctx context.Context, env cfg.Context, planner run
 				}
 			}
 
-			return planResources(ctx, secrets, planner, registry, stackAndDefs.Stack, rp)
+			return planResources(ctx, secrets, planner, registry, stackAndDefs.Stack, rp, ingressFragments)
```

Comment:
It would be best if `ingressFragments` are part of planned server stack.

I suggest introducing: 

```
package planning

type StackWithIngress struct {
  Stack
  IngressFragments []*schema.IngressFragment
}
```

And then construct an `StackWithIngress` here, and pass it instead of `stackAndDefs.Stack`.

`serverStack` required by `planResources` would be augmented with `GetIngresses(srv schema.PackageName) ([]*schema.IngressFragment, error)`, which `StackWithIngress` would implement.

Similarly the signature of `makeServerConfig` wouldn't change.

Reply:
Done.